need held while installing apollo (version 1.0.3) Via cocoapods ,getting error as follows.

Hi i am getting following error in terminal while installing Apollo version '1.0.3' via cocoapods in my ios swift application,
can anyone help me with this. Thanks 
---------------------------------------------------
pod install
Analyzing dependencies
Downloading dependencies
Installing Apollo (1.0.3)
[!] /bin/bash -c 
set -e
make clean build-cli-for-cocoapods
cp .build/release/apollo-ios-cli apollo-ios-cli
chmod +x apollo-ios-cli

swift package clean
swift build --product apollo-ios-cli -c release -Xswiftc -DCOCOAPODS
/Users/bhushandeshmukh/Library/Caches/CocoaPods/Pods/Release/Apollo/1.0.3-936d4: error: package at '/Users/bhushandeshmukh/Library/Caches/CocoaPods/Pods/Release/Apollo/1.0.3-936d4' is using Swift tools version 5.6.0 but the installed version is 5.5.0
make: *** [build-cli-for-cocoapods] Error 1
-----------------------------------------------------

--- a/Apollo.podspec
+++ b/Apollo.podspec
@@ -7,7 +7,7 @@ Pod::Spec.new do |s|
   s.summary = "A GraphQL client for iOS, written in Swift."
   s.source = { :git => 'https://github.com/apollographql/apollo-ios.git', :tag => s.version }
   s.requires_arc = true
-  s.swift_version = '5.0'
+  s.swift_version = '5.6'
   s.default_subspecs = 'Core'
   s.ios.deployment_target = '12.0'
   s.osx.deployment_target = '10.14'

